stylus/cozy-ui/build.styl

```diff
@@ -864,10 +901,6 @@ Display an chip that represents complex identity
  <h4 class="u-title-h4">Heading 4</h4>
  <p class="u-text">Basic text</p>
  <p class="u-caption">Caption text</p>
- <p class="u-t-left">Left aligned text</p>
- <p class="u-t-right">Right aligned text</p>
- <p class="u-t-center">Center aligned text</p>
- <p class="u-t-justify">Justify aligned text</p>
```

Comment:
are you removing theses classes?

Reply:
Are we sure they are not used within an app?

Reply:
Moving only, not removing. it wasn't in the right place.

Reply:
You renamed `u-t-justify` to `u-ta-justify`, no?

Reply:
Ah yes, you meant that. Indeed. I don't think they were used since they weren't exposed globally before. But I'll look around. It's not used in UI though.

Reply:
Confirmed, it's only used on the onboarding infra, which is the project i'm working on right now.